doc/syslog/syslog-design.md

```diff
@@ -430,10 +639,14 @@ _config syslog add_
 1. `-s|--source` - source ip address
 2. `-p|--port` - server udp port
 3. `-r|--vrf` - vrf device
+4. `--protocol` - server protocol
+5. `--filter_include` - filter include regex
```

Comment:
Please consider the command options as "--filter_type <include/exclude> --filter_regex <string> --severity <>".

Reply:
done